There has been a change in the method onResponse, in the section of code starting on line nr 138.
  
The change is in a catch statement, in a try statement, in the method ```onResponse```, in the class ```1```, in an initialization, in the invocation ```bulk```, in the method ```handleRequest```, in the class ```RestBulkAction```.
  
The following changes have been made:  
Changed type reference from ```java.lang.Exception``` to ```Throwable``` on line 138.  
